src/components/List/List.jsx

```diff
@@ -100,6 +135,9 @@ const List = () => {
                     }
                   />
                   {doc.data().name}
+                  <button onClick={() => handleDelete(doc.id)}>
```

Comment:
This button is invisible to screen readers. Can we either add an `aria-label` or add some hidden text associated with the button? 
Here is a great article about how to make icon buttons accessible: https://www.sarasoueidan.com/blog/accessible-icon-buttons/

Reply:
it has been solved!